Services/DataBaseService/Repositories/TradeRepository.cs

```diff
@@ -95,7 +95,7 @@ private void UpdatePortfolioAfterTransaction(Transaction transaction)
             {
                 var exception = new BadRequestException("Not enough instrument count to sell");
                 logger.LogWarning(exception,
-                    $"User {transaction.UserId} asked to sell more instruments {transaction.Figi} than he has");
+                    $"{Guid.NewGuid()}_User {transaction.UserId} asked to sell more instruments {transaction.Figi} than he has");
```

Comment:
Why do we need this guid?

Reply:
> Why do we need this guid?

This is made for ParentId in logs

Reply:
It is just the first line? We have another field for parent id. Why do we have to add id in the message?